core/src/test/scala/unit/kafka/utils/JaasTestUtils.scala

```diff
@@ -169,6 +169,18 @@ object JaasTestUtils {
     jaasFile
   }
 
+  // Returns a SASL/SCRAM configuration using credentials for the given user and password
+  def scramClientLoginModule(mechanism: String, scramUser: String, scramPassword: String): String = {
+    mechanism match {
+      case "SCRAM-SHA-256" | "SCRAM-SHA-512" =>
```

Comment:
Can we not have these be naked constants?  Maybe we could iterate over an enum or something?  If we add another mechanism, it would be nice not to have to manually edit this.

Reply:
I fixed this in the other places in this file as well